### ISSUE TITLE
Weird empty line entries after tenth new line

### ISSUE THREAD

**Issue by pushchris** [codeshifu/sync-dotenv#7]
No matter how I order the .env file always it always gets screwed up after the tenth extra new line. The following gets inserted instead of new lines: ```__EMPTYLINE_10__=``` ```__EMPTYLINE_11__=``` etc instead of a blank line. 

As a convoluted example:
```
APP_NAME=

APP_ENV=

APP_KEY=

APP_DEBUG=

APP_URL=

LOG_CHANNEL=

DB_CONNECTION=

DB_HOST=

DB_PORT=

DB_DATABASE=

DB_USERNAME=

DB_PASSWORD=

OTHER_ENV=
```
Gets synced as:
```
APP_NAME=

APP_ENV=

APP_KEY=

APP_DEBUG=

APP_URL=

LOG_CHANNEL=

DB_CONNECTION=

DB_HOST=

DB_PORT=

DB_DATABASE=
__EMPTYLINE_10__=
DB_USERNAME=
__EMPTYLINE_11__=
DB_PASSWORD=
__EMPTYLINE_12__=
```

Not sure if its on my end or the scripts end but was hoping you might have some answers

**Comment by codeshifu** [comment]
Thanks for reporting this issue @pushchris. I'm looking into it right away

**Comment by codeshifu** [comment]
@pushchris this issue has been fixed in [v2.0.2](https://github.com/codeshifu/sync-dotenv/releases/tag/v2.0.2)

**Comment by pushchris** [comment]
Love the quick turn around! Thank you @codeshifu
